### DISCUSSION TITLE
[Screen touch Support] Can't Use Touch screen after rofi is running

### DISCUSSION THREAD

**Discussion by awarewen** [davatorium/rofi#1880] [(Possible) Issues]
I use 'rofi' on a touchscreen device, but when rofi is running, the touch screen is completely unavailable and can only turn off 'rofi'.

-- Command
```
❯ lisgd -d /dev/input/event13 -g "1,TR,*,*,R,notify-send swiped lr && wofi --show run && wvkbd-mobintl -L 240"
```

Normally 'rofi' and 'wvkbd' should be turned on at the same time, and I can use the virtual keyboard for input, but sadly this is not possible.

**Comment by DaveDavenport** [comment]
I have tried rofi with touchscreen, but never with virtual keyboard. 

Two questions:
1.) Can you try out launching rofi with `-normal-window`, does this make a difference?
2.) Can you give a bit more information about your setup. (rofi version, desktop environment,  Xorg/Wayland, etc.)

**Reply by DaveDavenport** [reply]
wait, I only notice now..   You are using wofi?  wofi != rofi.

**Comment by DaveDavenport** [comment]
Wofi is not rofi, wrong discussion forum.

**Comment by awarewen** [comment]
## Sorry for the late reply
-- Command run: ` lisgd -d /dev/input/event13 -g "1,RL,*,*,R,notify-send next worksapce && rofi -show run && wvkbd-mobintl -L 300" ` 

https://github.com/davatorium/rofi/assets/53365857/7bf2a964-3080-43ca-8c94-7b30856c5d3f

Same like wofi.

**Reply by DaveDavenport** [reply]
https://github.com/lbonn/rofi/

**Reply by awarewen** [reply]
thanks, I will try it.

**Reply by DaveDavenport** [reply]
You are already running that version of rofi, giving your recording.

**Reply by awarewen** [reply]
https://github.com/davatorium/rofi/assets/53365857/3c2c152d-d4a5-4e51-bbed-74f9171dc123

❯ rofi -v
Version: 1.7.5+wayland2-1-gff2338c3 (makepkg)

pkg: rofi-lbonn-wayland-git 1.7.5.wayland2.r1.gff2338c3-1

❯ hyprctl version
Hyprland, built from branch main at commit 261c3307f70d14e5bbc7d4113b4ee6dac9e4b627 dirty (internal: cleanup headers in helpers/).
Tag: v0.28.0-11-g261c3307

flags: (if any)

**Comment by awarewen** [comment]
https://github.com/davatorium/rofi/assets/53365857/da1c35e9-f2e3-475e-97c1-544edb54e475

 I ran three commands, one is to swipe up the screen to open 'rofi' and 'wvkbd', and the other two control the workspace, but after starting 'rofi', the mouse click for 'waybar' fails and also cannot start 'wvkbd' at the same time, but then sliding left and right to change the workspace is effective.

**Comment by awarewen** [comment]
And the 'rofi' list itself doesn't respond to touch clicks or swipe up, which is not normal

**Comment by DaveDavenport** [comment]
rofi does not support wayland. I don't use it myself, I do not know how it works and I have no way to debug it.

It looks like you are not running this rofi, but the wayland port (linked above). 
You do not provide information (even after asked), I need to deduce you using wayland, another rofi version, etc from your screenshots/recordings.

**Reply by awarewen** [reply]
Sorry I brought this issue in the wrong place, the 'x11' version 'rofi' works with 'wvkbd', I just tried it. Thank you very much for your reply. :smile_cat:
